### PR TITLE
add spelling bee puzzle game

### PR DESCRIPTION
**Pull request by ConorSheehan1**

### CODE REVIEW

**Comment by ConorSheehan1** [comment]
My bad, I see I need to add the original game too. Will update tomorrow

File: games/s.yaml [hunk 1]
```diff
@@ -1038,6 +1038,26 @@
...
+  - Vue.js
+  licenses:
+  - MIT
+  development: active
+  originals:
+  - Spelling Bee
+  repo: https://github.com/ConorSheehan1/spelling-bee
+  status: playable
+  type: similar
+  content: free
```
Comment (cxong) [inline games/s.yaml:1058]:
```suggestion
```
I don't see any copyrightable content (art, music etc)

File: games/s.yaml [hunk 2]
```diff
@@ -1038,6 +1038,26 @@
...
+  frameworks:
+  - Vue.js
+  licenses:
+  - MIT
+  development: active
+  originals:
+  - Spelling Bee
+  repo: https://github.com/ConorSheehan1/spelling-bee
+  status: playable
+  type: similar
```
Comment (cxong) [inline games/s.yaml:1057]:
```suggestion
  type: clone
```

File: games/s.yaml [hunk 3]
```diff
@@ -1038,6 +1038,26 @@
   type: remake
   updated: 2015-04-05
   url: http://bringerp.free.fr/RE/Speedball2/remake.php5
+  
+- name: Spelling Bee
+  images:
+  - https://github.com/ConorSheehan1/spelling-bee/blob/main/.github/images/spelling-bee.png
+  - https://github.com/ConorSheehan1/spelling-bee/blob/main/.github/images/spelling-bee-dark.png
```
Comment (cxong) [inline games/s.yaml:1045]:
```suggestion
  - https://raw.githubusercontent.com/ConorSheehan1/spelling-bee/main/.github/images/spelling-bee.png
  - https://raw.githubusercontent.com/ConorSheehan1/spelling-bee/main/.github/images/spelling-bee-dark.png
```

**Review by cxong — CHANGES_REQUESTED** [review]

**Review by cxong — APPROVED** [review]

**Pull request merged**